tests/dracut_tests/test_driver_updates.py

```diff
@@ -433,10 +434,11 @@ class LoadDriversTestCase(unittest.TestCase):
     def test_basic(self, call):
         """load_drivers: runs depmod and modprobes all named modules"""
         modnames = ['mod1', 'mod2']
-        load_drivers({name: [name] for name in modnames})
+        moddict = collections.OrderedDict({name: [name] for name in modnames})
```

Comment:
Ignore this, solved on IRC.

In short, ``dict`` do a randomize output before but it doesn't matter because comparison is then still ordered in ``OrderedDict``.